src/components/App/App.js

```diff
@@ -41,7 +45,25 @@ class App extends Component {
       });
   }
 
-
+  searchMakeup = userInput => {
+    if(userInput !== '') {
+    const filteredMakeupTag = [];
+    this.state.makeup.forEach(item => {
+      const tags = item["tag_list"].map(tag => tag.toLowerCase());
+      if(tags.includes(userInput)) {
+        filteredMakeupTag.push(item)
+      }
+    })
+    const filterMakeupWithoutBrand = this.state.makeup.filter(item => item.brand)
```

Comment:
Thank you for adding this functionality! I'm looking forward to hearing your explanation of it.